lib/components/user/link-button.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Button } from 'react-bootstrap'
+
+import { routeTo } from '../../actions/ui'
```

Comment:
Please use import wildcard syntax.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/b4e99d7f6239a443ded667203754be15572ebf80